src/AppBundle/Repository/AutoEditsRepository.php

```diff
@@ -20,6 +20,22 @@ class AutoEditsRepository extends UserRepository
     /** @var array List of automated tools, used for fetching the tool list and filtering it. */
     private $aeTools;
 
+    /** @var bool Whether to use the /sandbox version of the config, bypassing caching. */
+    private $useSandbox;
+
+    /** @var array Process cache for tags/IDs. */
+    private $tags;
+
+    /**
+     * AutoEditsRepository constructor. Used solely to set $useSandbox (from AutomatedEditsController).
+     * @param bool $useSandbox
+     */
+    public function __construct(bool $useSandbox = false)
+    {
+        $this->useSandbox = $useSandbox;
+        parent::__construct();
```

Comment:
Not sure about within this repo, but generally in my experience the parent::__construct() call is put first